src/app/search/state/search.service.ts

```diff
@@ -108,11 +110,11 @@ export class SearchService {
    * @param {number} query_size
    * @memberof SearchService
    */
-  filterEntry(hits: Array<any>, query_size: number) {
+  filterEntry(hits: Array<any>, query_size: number): [Array<any>, Array<any>] {
```

Comment:
This is of type:
```
Array<{
            _index: string;
            _type: string;
            _id: string;
            _score: number;
            _source: T;
            _version?: number;
            _explanation?: Explanation;
            fields?: any;
            highlight?: any;
            inner_hits?: any;
            sort?: string[];
        }>
```
Might want to define this somewhere

Reply:
Defined an interface Hit and replaced Array<Any> with Array<Hit> in the filterEntry method and all associated methods.